core/classifiers/classifier_utils_test.py

```diff
@@ -245,3 +248,19 @@ def test_convert_float_numbers_to_string_in_classifier_data(self):
             classifier_utils.convert_float_numbers_to_string_in_classifier_data(
                 test_dict))
         self.assertDictEqual(expected_dict, output_dict)
+
+    def test_that_pretrained_models_for_all_classifier_are_correct(self):
+        """Make sure that trained classifier models generated in the output
+        by each classifier do not raise Exception when passed through
+        convert_float_numbers_to_string_in_classifier_data function."""
+
+        classifier_ids = (
+            algorithm_registry.Registry.get_all_classifier_algorithm_ids())
+        for classifier_id in classifier_ids:
+            file_path = os.path.join(
+                vmconf.PRETRAINED_MODELS_PATH, '%s.json' % classifier_id)
+            with open(file_path, 'r') as f:
+                classifier_data = json.loads(f.read())
+                output_dict = classifier_utils.convert_float_numbers_to_string_in_classifier_data( # pylint: disable=line-too-long
```

Comment:
Would be nice if this name was shorter, otherwise we'll keep running into this issue...

Maybe encode_classifier_data_for_transport or encode_floats_in_classifier_data or convert_floats_to_strings(classifier_data) or something like that?

Reply:
Changed to `encode_floats_in_classifier_data`.